Verbale 2025 03 03 (#162)
SevenBitsSwe/7BitsDocs: PB/Documentazione Esterna/Verbali Esterni/VerbaliEsterni-2025-03-03.tex

\end{itemize}

\subsection{Oggetto}
Incontro organizzativo con Andrea Dorigo e Fabio Pallaro dell'azienda SyncLab, finalizzato a pianificare lo sviluppo del progetto a seguito della consegna RTB, 
e allineare gruppo e azienda su quanto svolto fino ad ora.

\subsection{Sintesi}
Durante l'incontro la proponente è stata messa al corrente dello stato attuale dello sviluppo del progetto, e nello specifico dell'esito della consegna RTB ricevuto in data
2025-03-01. Il gruppo ha inoltre informato la proponente del cambiamento della data di consegna prevista per il progetto, pianificata ora per il 2025-04-04.\\
Ha seguito l'esposizione di quanto svolto fino ad ora in ambito di progettazione: nello specifico è stata presentata una bozza di architettura software per la componente
di simulazione dei sensori, che è stata approvata dall'azienda, con qualche indicazione sulle possibili librerie da usare per la gestione dei thread, oltre a quella 
standard.\\

\subsubsection{Obiettivi per la realizzazione dell'MVP}
La proponente ha inoltre richiesto al gruppo di definire in maniera precisa e tracciabile gli obiettivi da concretizzare con la realizzazione dell'MVP, e che differenzieranno

> quest'ultimo dal PoC fino a questo momento sviluppato dal team. Gli obiettivi individuati sono i seguenti:

Reviewer: ```suggestion
quest'ultimo dal PoC sviluppato dal team. Gli obiettivi individuati sono i seguenti:
```

\begin{itemize}
    \item Realizzazione di una dashboard specifica per utente che consenta la visualizzazione dei dati (messaggi generati e percorso seguito) accessibile dalla dashboard 
    principale;

>     \item Realizzazione di una tabella che per ogni punto di interesse registrato mostri il numero di messaggi generati;

Reviewer: ```suggestion
    \item Realizzazione di una tabella che per ogni punto di interesse registrato mostri il numero di messaggi generati nel mese corrente e ordini i punti di interesse in base a questo ultimo dato;
```
Specificherei che appunto la tabella è ordinata secondo questo dato e che sono riferiti al mese corrente. Sui requisiti stiamo sul preciso.

>     \item Aumento del numero di utenti che vengono simulati contemporaneamente fino a \simeq10 (attualmente 1).

Reviewer: ```suggestion
    \item Aumento del numero di utenti simulati contemporaneamente fino a \simeq10 (attualmente 1).
```

\end{itemize}

\subsection{Decisioni Prese}

> \'E stata deciso di ridurre la durata delle iterazioni da due settimane a una fino alla consegna RTB a partire dal prossimo sprint (che inizierà in data 2025-03-07); e 

Reviewer: ```suggestion
\'E stato deciso di ridurre la durata degli sprint da due settimane a una fino alla consegna PB. Questo cambiamento avverrà dal prossimo sprint, che inizierà in data 2025-03-07. 
```

> il gruppo si è posto l'obiettivo di completare e presentare la proposta di architettura per la componente di simulazione dei sensori per il prossimo SAL (fissato con 

Reviewer: ```suggestion
Il gruppo si è posto l'obiettivo di completare e presentare la proposta di architettura per la componente di simulazione dei sensori per il prossimo SAL (fissato con 
```
